quinn-proto/src/cid_queue.rs

```diff
@@ -94,9 +96,9 @@ impl CidQueue {
         self.buffer[self.cursor].unwrap().0
     }
 
-    /// Check whether self.offset points to a valid CID in CidQueue
-    pub fn is_active_retired(&mut self) -> bool {
-        self.buffer[self.cursor].is_none()
+    /// Return active reset token
+    pub fn active_reset_token(&self) -> Option<ResetToken> {
+        self.buffer[self.cursor].unwrap().1
```

Comment:
Why is this unwrap() safe? Could/should we `map()` instead?

Reply:
Mooted in refactoring.